dataplane/dplanerc/interface.go

```diff
@@ -1093,7 +1172,7 @@ func (ni *Reconciler) setupPorts(ctx context.Context) error {
 			Type:            saipb.RouterInterfaceType_ROUTER_INTERFACE_TYPE_PORT.Enum(),
 			PortId:          &portResp.Oid,
 			SrcMacAddress:   tap.Attrs().HardwareAddr,
-			VirtualRouterId: proto.Uint64(0),
+			VirtualRouterId: proto.Uint64(ni.niDetail[fakedevice.DefaultNetworkInstance].vrOID),
```

Comment:
same here.

Reply:
the DEFAULT network instance always exists, so this won't be nil